.github/workflows/manual_installer_all_platforms.yml

```diff
@@ -3,22 +3,22 @@ name: Test OS SDK Installer - All Platforms
 on:
   workflow_dispatch:
     inputs:
-      os_installer_ubuntu_1804:
-        description: 'Ubuntu 1804 (.deb) URL'
+      base_url:
+        description: 'The base url to use to download each platform'
```

Comment:
Do we expect to use builds from s3 instead of a prerelease? just curious

Reply:
yes, I recently used s3 as a test and works fine.